### PR TITLE
Seed Project Path Update

### PR DESCRIPTION
**Pull request by mehreencs87**

### CODE REVIEW

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

File: articles/server-platforms/nancyfx/01-login.md [hunk 1]
```diff
@@ -12,10 +12,11 @@ This tutorial and seed project have been tested with the following:
 :::
 
 <%= include('../../_includes/_package', {
-  pkgRepo: 'Auth0.NancyFx.SelfHost',
```
Comment (mpaktiti) [inline articles/server-platforms/nancyfx/01-login.md:15]: @mehreencs87 Hi, another change was introduced and we need to apply it before this PR is merged. `pkgOrg` was added:
- if the sample is under `auth0-samples` repo add the line: `pkgOrg: 'auth0-samples'`
- if the sample is under `auth0` repo add the line: `pkgOrg: 'auth0'`
  Details: https://github.com/auth0/docs/blob/master/CONTRIBUTING.md#seed-projects

Can you please update this? Thanks.

**Comment by mehreencs87** [comment]
Check the latest commit.

**Comment by mpaktiti** [comment]
@mehreencs87 The [download URL](https://auth0.com/docs/package/auth0-nacyfx-samples/master?org=auth0-samples&path=00-Starter-Seed&file_path=00-Starter-Seed/App.config&type=replace) gives an error:

```
{"status":500,"message":"ENOENT: no such file or directory, open '/tmp/githubdownload11679-15315-vcjkck/auth0-nacyfx-samples-master/00-Starter-Seed/App.config'","error":{"cause":{"errno":-2,"code":"ENOENT","syscall":"open","path":"/tmp/githubdownload11679-15315-vcjkck/auth0-nacyfx-samples-master/00-Starter-Seed/App.config"},"isOperational":true,"errno":-2,"code":"ENOENT","syscall":"open","path":"/tmp/githubdownload11679-15315-vcjkck/auth0-nacyfx-samples-master/00-Starter-Seed/App.config"}}
```

Probably something is wrong with the Packer. I will check this and get back to you.

**Comment by mpaktiti** [comment]
@mehreencs87 There is something wrong with the packager and the `type=replace` option. I registered [a bug](https://github.com/auth0/auth0-package-creator/issues/33). Until this is resolved can you please update the value of `type` to `none` so I can merge this PR? Thanks.

**Comment by mehreencs87** [comment]
Updated @mpaktiti .

**Comment by mpaktiti** [comment]
Fixed by https://github.com/auth0/docs/pull/2078

**Pull request closed without merging**